dispatch/apps/events/views.py

```diff
@@ -0,0 +1,50 @@
+from django.core.files.uploadedfile import SimpleUploadedFile
+from django.shortcuts import render, redirect
+from django.http import HttpResponse
+
+from dispatch.apps.events.facebook import FacebookEvent
+from dispatch.apps.events.forms import EventForm
+from dispatch.apps.events.models import Event
+
+def success(request):
+
+    return HttpResponse('Your event has been created')
+
+def submit(request):
+
+    if request.POST.get("submit_facebook") and (request.POST.get('facebook_url') is not None):
+        # show form with facebook event data
+
+        event = FacebookEvent(request.POST.get('facebook_url'))
+
+        event_data = event.get_json()
+        event_data['facebook_url'] = request.POST.get('facebook_url')
+        event_data['facebook_image_url'] = event.get_image()
```

Comment:
I think it might make more sense to handle all of this inside the `FacebookEvent` class, and then call it like so:

```py
event_data = event.get_data()

form = EventForm(initial=event_data)